### ISSUE TITLE
Conditional restrictions OSM Database

### ISSUE THREAD

**Issue by JuanMeloColombia** [organicmaps/organicmaps#9746]
In Colombia, we have been working in OSM to implement mobility restrictions due to Ciclovías, which are open on Sundays and public holidays in cities like Bogotá, Medellín, and Villavicencio. This year, similar restrictions were also added in São Paulo, Brazil, for the Ciclovía on Avenida Paulista. The goal is for OSM-based routers to generate more efficient routes by avoiding temporarily closed roads and helping drivers find optimal routes that reflect real traffic conditions.

However, tests conducted with Organic Maps on Sundays in Bogotá show that roads closed between 7:00 a.m. and 2:00 p.m. due to the Ciclovía are still being considered in the routes. This indicates that conditional mobility restrictions are not being processed correctly.

It is crucial for routers to interpret and integrate these restrictions to ensure their accuracy and efficiency during the specific hours of the Ciclovías, which vary by city. How can we ensure that Organic Maps properly interprets and applies these restrictions?

To learn about the process for creating Ciclovías, conditional restrictions, and the information used, this guide was created on the Wiki: [wiki.openstreetmap.org/wiki/ES:Colombia/Proyecto-Ciclovías/Bogotá](https://wiki.openstreetmap.org/wiki/ES:Colombia/Proyecto-Ciclov%C3%ADas/Bogot%C3%A1). Similar guides are also available for Medellín and Villavicencio in Colombia.

some highways that can be taken as an examples
https://www.openstreetmap.org/way/910796773#map=18/4.657903/-74.118952
https://www.openstreetmap.org/way/375277498#map=18/4.622384/-74.066783
https://www.openstreetmap.org/relation/14183166#map=18/4.622266/-74.066665
https://www.openstreetmap.org/way/1076084920

**Comment by map-per** [comment]
It's a duplicate of #3731

**Comment by JuanMeloColombia** [comment]
In the Telegram group, your people recomended to create this one because the previus one didnt include bicycle conditional restrictions.
But if you say that it is solve its ok, I'll test

**Comment by vng** [comment]
https://github.com/organicmaps/organicmaps/pull/11624

Should fix ```bicycle:conditional```.

```oneway:conditional``` and ```oneway:bicycle:conditional``` in plans.

**Comment by JuanMeloColombia** [comment]
Indeed, OrganicMaps now correctly interprets the bicycle:conditional tag, meaning the router understands that bicycles can use the road on Sundays and public holidays.

However, the issue persists with the vehicle (car) profile: when using navigation in car mode, the router still allows routes through these roads, even though in OSM they are tagged with motor_vehicle:conditional=no @ (Su,PH 07:00-14:00).

This causes an error, since those roads are closed to motorized traffic during that time and are exclusively open for bicycles.

**Comment by vng** [comment]
@JuanMeloColombia Did you check it in the latest Beta? motor_vehicle:conditional should be ok there.

If you check the Store app version, the changes will take effect in the next public release.
